lang/en.js

```diff
@@ -39,7 +39,7 @@ export default {
 	"components.input-time-range.errorBadInput": "{startLabel} must be before {endLabel}",
 	"components.input-time-range.startTime": "Start Time",
 	"components.list-item-drag-handle.default": "Reorder item action for {name}",
-	"components.list-item-drag-handle.keyboard": "Reorder items, current position {currentPosition} out of {size}",
+	"components.list-item-drag-handle.keyboard": "Reorder item, current position {currentPosition} out of {size}. To move this activity, press up or down arrows.",
```

Comment:
Oh, one sec... Are you sure you want "activity" baked into this message?  Seems like this should be "item".

Reply:
Yeah good point, will double check with Abbie